.github/executables/checkAllSpecs

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+from concurrent.futures import ThreadPoolExecutor
+import glob
+import json
+import os
+import re
+import subprocess
+import sys
+from threading import Lock
+
+outputMutex = Lock()
+
+# Exclude config files that are not verifying from CI
+EXCLUDE_CONFS = [
```

Comment:
These are excluded temporarily because they do not yet pass verification again after the code updates. I have work in flight to fix 2 of these as separate PRs. I figured it is best to fix these as separate PRs because doing so will result in smaller self-contained changes for code reviewers to look at, and the updates may warrant discussion.

Reply:
Resolving